.circleci/config.yml

```diff
@@ -3,18 +3,18 @@ jobs:
   build:
     working_directory: ~/wagtailquickcreate
     docker:
-      - image: circleci/python:3.8.0
+      - image: cimg/python:3.12.1
         environment:
           PIPENV_VENV_IN_PROJECT: true
           DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
-      - image: circleci/postgres:9.6.2
+      - image: cimg/postgres:16.2.0
```

Comment:
**issue:** CI is currently failing because this image isn't found. Looking at [the list of available images](https://circleci.com/developer/images/image/cimg/postgres), I think this should fix it:

```suggestion
      - image: cimg/postgres:16.2
```